cmd/facts.go

```diff
@@ -81,17 +81,29 @@ func gather(*cobra.Command, []string) {
 	var argument = viper.GetString("argument")
 	var pluginsFolder = viper.GetString("plugins-folder")
 
-	engine := factsengine.NewFactsEngine("", "")
+	gathererManager := gatherers.NewManager(gatherers.StandardGatherers())
```

Comment:
Maybe we could have a `gatherers.NewManagerWithPlugins(gatherers.StandardGatherers(), pluginsFolder` to remove some duplicated code